build-tools/packages/build-cli/src/commands/generate/changeset.ts

```diff
@@ -285,9 +285,6 @@ export default class GenerateChangesetCommand extends BaseCommand<typeof Generat
 		);
 		const changesetPath = path.relative(context.gitRepo.resolvedRoot, newFile);
 
-		// Update the UPCOMING.md file
-		await GenerateUpcomingCommand.run(["--releaseGroup", "client", "--releaseType", bumpType]);
```

Comment:
wouldn't we still want this so release oce can generate the roll up at release time?

Reply:
This change removes the automatic/implicit generation of UPCOMING.md when a changeset is created using the CLI. If left in place, UPCOMING.md would continue to automatically generate each time a new changeset is created.